src/sub/tests/conftest.py

```diff
@@ -27,9 +31,10 @@
 
 logger = get_logger()
 
-ddb_process = None
 THIS_PATH = os.path.join(os.path.realpath(os.path.dirname(__file__)))
 UID = str(uuid.uuid4())
+IMAGE = "vitarn/dynalite:latest"
+CONTAINERS_FOR_TESTING_LABEL = "pytest_docker_log"
```

Comment:
seems like another hardcoded configuration value; on top of that the variable and value naming are unclear

Reply:
Why "containers" plural?

Reply:
resolved.